howto/install/customise.md

```diff
@@ -179,14 +53,9 @@ anbox-stream-gateway:
   to: ['0']
 ```
 
-There are a few entries under `options`, in this case to enable the development UI and point Prometheus at a specific port. There are however, a number of configuration options available (more details are in [the charm documentation](https://jaas.ai/u/anbox-charmers/anbox-stream-gateway/64)). We can add additional configuration by supplying the desired settings under options. So, for example, where we might do the following through Juju to set some proxy values:
-
-```bash
-$ juju config anbox-stream-gateway https_proxy=https://proxy.example.com
-$ juju config anbox-stream-gateway snap_proxy=https://snap-proxy.example.com
-```
+The entries under `options` in this example enable the web dashboard and assign a specific port for Prometheus. There are, however, a number of additional configuration options available (see the [charm documentation](https://jaas.ai/u/anbox-charmers/anbox-stream-gateway/64)). To add further configurations, specify the desired settings under `options`.
```

Comment:
Actually, the options enable the **Dev UI**, with a configuration option that is supposed to be ignored. But it is still set in the anbox-cloud bundle. Does that example here still make sense, or should we update it (and if so, how)?

Reply:
We can remove this option. We should also remove this option from the bundle, while harmless it might create confusion